components/bluetooth/lib.rs

```diff
@@ -664,10 +671,12 @@ impl BluetoothManager {
         }
     }
 
+    // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothremotegattserver-getprimaryservice
     fn get_primary_service(&mut self,
                            device_id: String,
                            uuid: String,
                            sender: IpcSender<BluetoothResponseResult>) {
+        // Step 2.5.
```

Comment:
These numbers don't correspond to step numbers in the spec, given the split between https://webbluetoothcg.github.io/web-bluetooth/#getgattchildren and https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothremotegattserver-getprimaryservice .